weather_dl/download_pipeline/manifest.py

```diff
@@ -26,7 +26,7 @@
 import traceback
 import typing as t
 
-from .util import to_json_serializable_type, fetch_geo_polygon, get_file_size
+from .util import to_json_serializable_type, fetch_geo_polygon, get_file_size, retry_with_exponential_backoff
```

Comment:
Since this line is getting a bit long, let's refactor it to use a multi-line import with parens: 
```suggestion
from .util import (
    to_json_serializable_type, 
    fetch_geo_polygon, 
    get_file_size, 
    retry_with_exponential_backoff
)
```